### DISCUSSION TITLE
Bedrock map of Antartica

### DISCUSSION THREAD

**Discussion by alexiemillikin** [earthlab-education/Earth-Analytics-2023-01-Intro#138] [Awesome science]
During an internship I worked on digitizing published bedrock geologic maps and geomorphic features from satellite imagery of Marie Byrd Land and the Ellsworth Mountains of Antarctica. These data are part of GeoMAP Antarctica, the first open-access geologic database of the frozen continent! GeoMAP was published earlier this year and is a collaborative effort with researchers from over 14 nations. 

[Article](https://www.gns.cri.nz/news/new-map-unlocks-deep-digital-data-of-antarcticas-history/?utm_source=Website&utm_medium=TW&utm_id=AntMAP80523-TW)
[Webviewer](https://data.gns.cri.nz/mapservice/apps/ata_geomap/index.html?map=Antarctic_v2022)

**Comment by cmq879** [comment]
Cool project and map. I'm curious about how you did the digitizing - were you using ArcGIS or another software?  I don't know much about bedrock or methods for categorizing it. Did you have to deal with any discrepancies across the sources you were compiling? From the article it sounds like the compilation of the different maps and records was a key part of the project.
